src/layouts/blog.js

```diff
@@ -0,0 +1,20 @@
+import React, { Fragment } from 'react'
+
+import Banner from '../components/banner'
+import BlogNav from '../components/blog/blog-navbar'
+import Footer from '../components/footer'
+
+
+export default ({ children }) => (
+  <Fragment>
+    {/* <Banner /> */}
```

Comment:
are we not using it?

Reply:
The banner is not currently on the blog site, but I wasn't sure if we wanted to keep it that way moving forward. Should I double check with DJ for how we want to handle this moving forward?

Reply:
Yea, DJ would have more info on that